ui/src/components/SampleControls/VideoSizeControl.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Dropdown } from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { setVideoSize } from '../../actions/sampleview';
+import styles from './SampleControls.module.css';
+
+function VideoSizeControl() {
+  const dispatch = useDispatch();
+
+  const videoSizes = useSelector((state) => state.sampleview.videoSizes);
+  const width = useSelector((state) => state.sampleview.width);
+
+  return (
+    <Dropdown
+      drop="down-centered"
+      onSelect={(i) => dispatch(setVideoSize(...videoSizes[i]))}
+    >
+      <Dropdown.Toggle className={styles.dropdownBtn} data-default-styles>
+        <i className={`${styles.controlIcon} fas fa-video`} />
+        <i className={`${styles.dropdownIcon} fas fa-sort-down`} />
+        <span className={styles.controlLabel}>Video size</span>
+      </Dropdown.Toggle>
+
+      <Dropdown.Menu className={styles.dropdownMenu}>
+        {videoSizes.map(([w, h], i) => {
+          const isActive = w.toString() === width;
```

Comment:
The sizes are defined as numbers in `sampleview.videoSizes`, but as strings in `sampleview.width/height`. It's not ideal. I traced back the latter to `HWR.beamline.sample_view.camera.get_width/height()`. Not sure what the feasibility would be of changing these methods to return numbers.